client/blocks/eligibility-warnings/index.jsx

```diff
@@ -161,9 +185,14 @@ EligibilityWarnings.defaultProps = {
 
 const mapStateToProps = state => {
 	const siteId = getSelectedSiteId( state );
+	const eligibilityData = getEligibility( state, siteId );
+	const dataLoaded = !! eligibilityData.lastUpdate;
```

Comment:
nice use of this here. I had personally envisioned a time aspect as well, for example, making sure that the eligibility information was at least as recent as X minutes

this may also be handled at the schema validation on load too in which case there might be no need to verify here either